dietpi/misc/dietpi-vpn

```diff
@@ -0,0 +1,469 @@
+#!/bin/bash
+{
+	#////////////////////////////////////
+	# DietPi VPN
+	#
+	#////////////////////////////////////
+	# Created by Daniel Knight / daniel.knight@dietpi.com / dietpi.com
+	#
+	#////////////////////////////////////
+	#
+	# Info:
+	# - GUI to setup a NordVPN or ProtonVPN connection
+	# - Allows to automatically connect on boot
+	# - Allows to add post-up and pre-down scripts
+	readonly USAGE='
+Usage: dietpi-vpn [<command>]
+Available commands:
+  <empty>		Interactive menu to edit, connect and apply VPN settings
+  status		Print VPN connection status info
+'	#////////////////////////////////////
+
+	# Grab Inputs
+	INPUT=$1
+
+	# Import DietPi-Globals --------------------------------------------------------------
+	# - Only load for interactive menu
+	if [[ $INPUT ]]; then
+
+		readonly G_PROGRAM_NAME='DietPi-VPN'
+
+	else
+
+		. /boot/dietpi/func/dietpi-globals
+		readonly G_PROGRAM_NAME='DietPi-VPN'
+		G_CHECK_ROOT_USER
+		G_CHECK_ROOTFS_RW
+		G_INIT
+
+	fi
+	# Import DietPi-Globals --------------------------------------------------------------
+
+	readonly FP_SETTINGS_DIETPI='/var/lib/dietpi/dietpi-vpn/settings_dietpi.conf'
+	readonly FP_SETTINGS_OVPN='/var/lib/dietpi/dietpi-vpn/settings_ovpn.conf'
+	readonly FP_SETTINGS_UP='/var/lib/dietpi/dietpi-vpn/up.sh'
+	readonly FP_SETTINGS_DOWN='/var/lib/dietpi/dietpi-vpn/down.sh'
+	readonly KILLSWITCH_UP='/var/lib/dietpi/dietpi-vpn/ks-up.sh'
+	readonly KILLSWITCH_DOWN='/var/lib/dietpi/dietpi-vpn/ks-down.sh'
+	readonly PROTONVPN_SERVERS='/etc/openvpn/protonvpn/serverlist.txt'
+	readonly CLIENT_FILE='/etc/openvpn/client.ovpn'
+	readonly IFACE='tun0'
+	PROTONVPN_INSTALLED=$([[ -d /etc/openvpn/protonvpn ]] && echo 1 || echo 0)
+	NORDVPN_INSTALLED=$([[ -d /etc/openvpn/nordvpn ]] && echo 1 || echo 0)
+	readonly MAX_WAIT_FOR_CONNECTION=10 # seconds
+	VPN_USERNAME=
+	VPN_PASSWORD=
+	VPN_SERVER=
+	PROTOCOL='udp'
+	VPN_CONNECTED=0
+	WAN_IP=
+	RX='N/A'
+	TX='N/A'
+
+	Update_Wan_Ip(){ WAN_IP=$(curl -4sSfLm 2 https://ifconfig.co/json 2>&1 | mawk -F\" '/"ip"/{o=$4}/"country"/{c=$4}/"region_name"/{c=$4"-"c}END{if(c){o=o" "c}if(o){print o;exit}{print}}'); }
```

Comment:
Is it a specific reason why IPv4 is forced for the IP? I think it's fine to show the IPv6 address when this is what is used by default.